mos-platform/atari8-stdcart/copy-data.c

```diff
@@ -0,0 +1 @@
+../common/crt0/copy-data.c
```

Comment:
We can't check symlinks into the SDK, or it won't build on Windows. Instead, just use this path in the CMakeLists.txt to atari8-stdcart; the oddness of that should be a signal that this needs cleaned up too.

Reply:
good point